### PR TITLE
rename returntocorp/semgrep to semgrep/semgrep for both Github.com an…

### PR DESCRIPTION
**Pull request by DrewDennison**
…d Docker references

# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [ ] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs
- [ ] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

### CODE REVIEW

File: docs/contributing/adding-a-language.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ in there, [talk to us](/support/).
 Otherwise, let's get started.
 
 Repositories involved directly:
-* [semgrep](https://github.com/returntocorp/semgrep): the semgrep command line program;
+* [semgrep](https://github.com/semgrep/semgrep): the semgrep command line program;
 * [ocaml-tree-sitter-semgrep](https://github.com/returntocorp/ocaml-tree-sitter-semgrep): language-specific setup, generates C/OCaml parsers for semgrep;
```
Comment (zzeleznick) [inline docs/contributing/adding-a-language.md:19]:
```suggestion
* [ocaml-tree-sitter-semgrep](https://github.com/semgrep/ocaml-tree-sitter-semgrep): language-specific setup, generates C/OCaml parsers for semgrep;
```

File: docs/contributing/adding-a-language.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ repository](https://github.com/returntocorp/semgrep):
 When done with the work in [ocaml-tree-sitter-semgrep](https://github.com/returntocorp/ocaml-tree-sitter-semgrep), you'll need a new repo semgrep-X to host the generated parser code.
 Ask someone from the Semgrep team to create one for you. For this, they should use
 the template
-[semgrep-lang-template](https://github.com/returntocorp/semgrep-lang-template)
+[semgrep-lang-template](https://github.com/semgrep/semgrep-lang-template)
 when creating the repo.
 
 The instructions for adding a language start in [ocaml-tree-sitter-semgrep](https://github.com/returntocorp/ocaml-tree-sitter-semgrep) (as indicated below). Be careful that you are always in the correct repo!
```
Comment (zzeleznick) [inline docs/contributing/adding-a-language.md:45]:
```suggestion
The instructions for adding a language start in [ocaml-tree-sitter-semgrep](https://github.com/semgrep/ocaml-tree-sitter-semgrep) (as indicated below). Be careful that you are always in the correct repo!
```

**Review by zzeleznick — APPROVED** [review]
LGTM w/ a few missed replacements

**Comment by DrewDennison** [comment]
finished some more in https://github.com/semgrep/semgrep-docs/pull/1363

**Pull request merged**